bindings/pyroot/pythonizations/test/dependency_versions.py

```diff
@@ -17,12 +17,10 @@
 ignore.append('distributed')
 ignore.append('dask-jobqueue')
 
-if sys.version_info[0] == 2 and 'ROOTTEST_IGNORE_NUMBA_PY2' in os.environ or \
-   sys.version_info[0] == 3 and 'ROOTTEST_IGNORE_NUMBA_PY3' in os.environ:
+if 'ROOTTEST_IGNORE_NUMBA_PY3' in os.environ:
     ignore += ['numba', 'cffi']
 
-if sys.version_info[0] == 2 and 'ROOTTEST_IGNORE_JUPYTER_PY2' in os.environ or \
-   sys.version_info[0] == 3 and 'ROOTTEST_IGNORE_JUPYTER_PY3' in os.environ:
+if 'ROOTTEST_IGNORE_JUPYTER_PY3' in os.environ:
```

Comment:
```suggestion
if 'ROOTTEST_IGNORE_JUPYTER' in os.environ:
```
Maybe that's also a matter of preference, but if we only support Python 3 anyway, do we need Python 3 suffixes in these variables?

Reply:
Namely the environment variables are `ROOTTEST_IGNORE_JUPYTER` and `ROOTTEST_IGNORE_NUMBA`